services/app-api/handlers/otel_handler.ts

```diff
@@ -0,0 +1,22 @@
+import opentelemetry from "@opentelemetry/api";
+import { NodeTracerProvider } from "@opentelemetry/sdk-trace-node";
+import { Resource } from "@opentelemetry/resources";
+import { OTLPTraceExporter} from "@opentelemetry/exporter-trace-otlp-grpc"
+import { SimpleSpanProcessor, ConsoleSpanExporter } from "@opentelemetry/sdk-trace-base";
+
+process.env.NODE_TLS_REJECT_UNAUTHORIZED = '0';
+
+const provider = new NodeTracerProvider({
+    resource: Resource.default().merge(new Resource({
+        "service.name": "mc-review",
+      })),
+})
+
+// log to console and send to New Relic
+provider.addSpanProcessor(new SimpleSpanProcessor(new ConsoleSpanExporter()))
```

Comment:
We might not totally care about the console exporter for the merge, since I feel like it's mainly good for debugging.

Or maybe we just comment it out with some instructions that say this line can be enabled for debugging?

Reply:
I think we absolutely want it enabled in local dev by default, I think this would be a good env var toggled setting

Reply:
I think that's cool for now, but I'd vote for local dev that we setup a container that is running [jaeger](https://www.jaegertracing.io/) and we point traces there in local. That way we can lean on the OTEL backend to start correlating the spans to a trace and not have to parse through console output.

Reply:
oh yes please. I don’t know what Jaeger is but i’d love for us to have local monitoring running